cz.zeropage.Formiko.json

```diff
@@ -0,0 +1,300 @@
+{
+    "app-id": "org.flatpak.Formiko",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.34",
+    "sdk": "org.gnome.Sdk",
+    "command": "formiko",
+    "finish-args" : [
+        "--share=ipc",
+        "--socket=x11",
+        "--socket=wayland",
+        "--socket=session-bus",
+        "--filesystem=home",
+        "--filesystem=/media",
+        "--filesystem=/keybase",
+        "--talk-name=org.gtk.vfs.*"
+    ],
+    "build-options" :
+    {
+        "cflags": "-O2 -g",
+        "cxxflags": "-O2 -g",
+        "env":
+        {
+            "V": "1"
+        }
+    },
+    "cleanup": [
+        "/include",
+        "/lib/pkgconfig",
+        "/share/pkgconfig",
+        "/share/aclocal",
+        "/man",
+        "/share/man",
+        "/share/gtk-doc",
+        "/share/vala",
+        "*.la",
+        "*.a"
+    ],
+    "modules": [
+        {
+            "name": "gtksourceview",
+            "config-opts": [ "--disable-Werror" ],
+            "sources": [
+            {
+                "type": "archive",
+                "url": "https://download.gnome.org/sources/gtksourceview/3.24/gtksourceview-3.24.11.tar.xz",
+                "sha256": "691b074a37b2a307f7f48edc5b8c7afa7301709be56378ccf9cc9735909077fd"
+            }
+            ]
+        },
+        {
+            "name": "intltool",
```

Comment:
It's available in https://github.com/flathub/shared-modules

Reply:
redefined